configuration/forms.md

```diff
@@ -496,3 +494,31 @@ To configure a form to send using Medic's custom SMS definition, add the field `
 ##### SMS content
 
 	U5 DANGER LCIB
+
+## XForms for Medic Collect
+
+ODK XForms are used to render forms in the Medic Collect Android app. These forms cannot use any Medic-specific notations described above. All Medic Collect forms are processed as SMS (even when submitted over a wifi) therefore a corresponding JSON form with matching fields is used to interpret the incoming report.
+
+Collect forms must be in the `forms/collect` folder to be processed by [`medic-conf`](https://github.com/medic/medic-conf)'s `convert-collect-forms` and `upload-collect-forms` actions. Once uploaded to the server, they can be downloaded by the Medic Collect app. These forms can also be included in Medic Collect builds for users without a data connection to get forms.
+
+XForms require a couple minor changes to be compatible with Medic Collect so that they can properly be received by a Medic instance. The changes can be done either manually in the XForm's XML, or automatically with XLSForm forms.
+
+### Manual changes in XForm
```

Comment:
I think we need to highlight this section more prominently.

Reply:
Are you referring to the Medic Collect section in general, or what needs to be done manually if you aren't using XLSForms and medic-conf?

Reply:
The section that has to do with the manual changes